sqlite3_opt_icu.go

```diff
@@ -8,10 +8,12 @@
 package sqlite3
 
 /*
-#cgo LDFLAGS: -licuuc -licui18n
+#cgo LDFLAGS: -licuuc
 #cgo CFLAGS: -DSQLITE_ENABLE_ICU
+#cgo linux LDFLAGS: -licui18n
 #cgo darwin CFLAGS: -I/usr/local/opt/icu4c/include
-#cgo darwin LDFLAGS: -L/usr/local/opt/icu4c/lib
-#cgo openbsd LDFLAGS: -lsqlite3
+#cgo darwin LDFLAGS: -L/usr/local/opt/icu4c/lib -licui18n
+#cgo openbsd LDFLAGS: -lsqlite3 -licui18n
```

Comment:
@mattn Do you know why we are using `-lsqlite3` here? I would think that would cause issues since we'd be simultaneously dynamic linking against the system and statically compiling our own version.

Reply:
Seems to be related on https://github.com/mattn/go-sqlite3/pull/595

Reply:
Hmm, I cannot test this since I don't have access to an openbsd system. And it's not clear if it was intentional. I will file a separate issue to investigate.